nixos/modules/services/mail/rss2email.nix

```diff
@@ -93,9 +93,11 @@ in {
 
     services.rss2email.config.to = cfg.to;
 
-    systemd.tmpfiles.rules = [
-      "d /var/rss2email 0700 rss2email rss2email - -"
-    ];
+    system.activationScripts.rss2email = lib.stringAfter [ "users" ] ''
+      if [ -e /var/rss2email -a ! -e /var/lib/rss2email ]; then
+          mv /var/rss2email /var/lib/rss2email
```

Comment:
I suppose this script can be optionally included depending on state version?
The tmpfiles rules line above needs to be changed to `/var/lib/rss2mail` because new users won't have this directory yet.

Reply:
Yeah, this should be an optionalString and gated on stateVersion. I'd prefer having it inside `preStart` too.